react/features/invite/components/InviteButton.native.js

```diff
@@ -144,19 +147,11 @@ function _mapStateToProps(state) {
     return {
         /**
          * Whether or not the feature to directly invite people into the
-         * conference is available.
-         *
-         * @type {boolean}
-         */
-        _addPeopleEnabled: isAddPeopleEnabled(state),
-
-        /**
-         * Whether or not the feature to dial out to number to join the
-         * conference is available.
+         * conference (via an external service or a phone number) is available.
          *
          * @type {boolean}
          */
-        _dialOutEnabled: isDialOutEnabled(state)
+        _addPeopleEnabled: isAddPeopleEnabled(state) || isDialOutEnabled(state)
```

Comment:
I decided to do this because there is no independent logic to each of them, and we want to do the same thing for both: open the add people dialog.